libraries/testing-tools/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { CompatServiceDefinition } from 'nice-grpc/lib/service-definitions';
+import { Client, createChannel, createClientFactory } from 'nice-grpc';
+import { getModuleNameInterceptor } from '@conduitplatform/grpc-sdk/dist/interceptors';
+import { IRRunDependenciesInterface } from './interfaces/IRRunDependenciesInterface';
+import { ChildProcess } from 'child_process';
+
+const { exec } = require('child_process');
+
+export default class TestingTools<T extends CompatServiceDefinition> {
+  _client: Client<T>;
+
+  constructor(private readonly serverAddress: string, serviceDefinition: T) {
+    const channel = createChannel(this.serverAddress, undefined, {
+      'grpc.max _receive_message_length': 1024 * 1024 * 100,
+      'grpc.max_send_message_length': 1024 * 1024 * 100,
+    });
+    const clientFactory = createClientFactory().use(getModuleNameInterceptor('test'));
+    this._client = clientFactory.create(serviceDefinition, channel);
+  }
+
+  async startRedis() {
+    exec('docker run --name conduit-redis -d -p 6379:6379 redis:latest');
```

Comment:
Add `REDIS_PORT` env for configuring the exported port and default to something else.
Otherwise running any of these tests locally while working with Conduit would require constantly shutting down our dev Redis.

This also doesn't just **start** Redis, it creates it too.
Change `conduit-redis` to `conduit-redis-tests` or something that shouldn't normally collide with anything else.
Depending on how these functions are used we may need to check whether the container already exists and start it instead as otherwise we'd be constantly recreating containers.